app/models/profile.rb

```diff
@@ -191,6 +191,20 @@ def deactivate(reason)
     update!(active: false, deactivation_reason: reason)
   end
 
+  # Update the profile's deactivation reason to "encryption_error". As a
+  # side-effect, this will also update the associated pending in-person
```

Comment:
Non-blocking nit: Maybe rephrase this comment to indicate that the profile won't necessarily have an `in_person_enrollment` associated with it?